meilisearch/index.py

```diff
@@ -267,7 +267,7 @@ def get_documents(self, parameters: Optional[Dict[str, Any]] = None) -> List[Dic
         Parameters
         ----------
         parameters (optional):
-            parameters accepted by the get documents route: https://docs.meilisearch.com/reference/api/documents.html#get-all-documents
+            parameters accepted by the get documents route: https://docs.meilisearch.com/reference/api/documents.html#get-documents
```

Comment:
The return definition should be updated. It does not return a list of dictionnaires anymore